its/projects/CSharp.SDK.2.1/AspNetCoreMvc/Startup.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace AspNetCoreMvc
+{
+    public class Startup
+    {
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
To make it work, see csproj
```suggestion
        public void ConfigureServices(IServiceCollection services) =>
            services.AddMvc();
```